src/components/FormValidator.js

```diff
@@ -56,10 +56,13 @@ export default class FormValidator {
     });
   }
 
```

Comment:
Please, add `this._toggleSubmitButton();`  in line 51 in `enableValidation` to initially disable the button on start


```js

  enableValidation() {
    this._toggleSubmitButton();  // here

    this._inputList.forEach((inputElement) => {
      inputElement.addEventListener("input", (evt) => {
```

Reply:
I decided to disable the button via HTML and add the appropriate CSS class to keep things consistent with the other forms. Hope that's ok too.